### ISSUE TITLE
Unable to read dependencies after updating to 0.11.1 

### ISSUE THREAD

**Issue by SekibOmazic** [jspm/jspm-cli#462]
After updating to 0.11.1 I can't create an application from scratch. I've tried like this:

`jspm init`

Output:

```Shell
Package.json file does not exist, create it? [yes]: 
Would you like jspm to prefix the jspm package.json properties under jspm? [yes]: 
Enter a name for the project (optional): 
Enter baseURL (public folder) path [.]: 
Enter project code folder [./lib]: 
Enter jspm packages folder [./jspm_packages]: 
Enter config file path [./config.js]: 
Configuration file config.js doesn't exist, create it? [yes]: 
ok   The app/* path has been set to lib/*.js.
     To alter this path, set the directories.lib in the package.json or run jspm init -p to set the code folder.


Which ES6 transpiler would you like to use, Traceur or 6to5? [traceur]: 
ok   Verified package.json at package.json
     Verified config file at config.js
     Looking up loader files...
       traceur.js
       traceur.js.map
       traceur.src.js
       system.js
       system.src.js
       system.js.map
       es6-module-loader.js
       es6-module-loader.src.js
       es6-module-loader.js.map
       traceur-runtime.js
       traceur-runtime.src.js
       traceur-runtime.js.map
     
     Using loader versions:
       es6-module-loader@0.13.0
       systemjs@0.13.0
       traceur@0.0.82
ok   Loader files downloaded successfully
```

After installing few dependencies (angular, bootstrap, ...) and loading the index.html in browser I get following in the browser's console:
![screen shot 2015-02-01 at 18 39 21](https://cloud.githubusercontent.com/assets/3735902/5993016/d46ba822-aa41-11e4-8b7a-6c218f5ecc19.png)

It is the same behavior no matter if using traceur or 6to5.

**Comment by guybedford** [comment]
This sounds like it may be an issue with your configuration - one of your dependencies might not be configured with angular as a dependency properly, so it doesn't know where to find it.

**Comment by guybedford** [comment]
Can you share your configuration file?

**Comment by SekibOmazic** [comment]
`config.js` (created by jspm):

```Javascript
System.config({
  "paths": {
    "*": "*.js",
    "app/*": "lib/*.js",
    "github:*": "jspm_packages/github/*.js"
  },
  "transpiler": "6to5"
});

System.config({
  "map": {
    "angular-animate@1.3.8": "github:angular/bower-angular-animate@1.3.8",
    "angular-route@1.3.8": "github:angular/bower-angular-route@1.3.8",
    "angular@1.3.8": "github:angular/bower-angular@1.3.8",
    "bootstrap": "github:twbs/bootstrap@3.3.2",
    "css@0.1.0": "github:systemjs/plugin-css@0.1.0",
    "github:angular/bower-angular-animate@1.3.8": {
      "angular": "github:angular/bower-angular@1.3.8"
    },
    "github:angular/bower-angular-route@1.3.8": {
      "angular": "github:angular/bower-angular@1.3.8"
    },
    "github:marcorinck/angular-growl@0.4.0": {
      "angular": "github:angular/bower-angular@1.3.8"
    },
    "github:twbs/bootstrap@3.3.2": {
      "css": "github:systemjs/plugin-css@0.1.0",
      "jquery": "github:components/jquery@2.1.3"
    }
  }
});
```

**Comment by guybedford** [comment]
What did you type to do your installs?

**Comment by SekibOmazic** [comment]
```Shell
jspm install angular@1.3.8 angular-route@1.3.8 angular-animate@1.3.8
jspm install github:marcorinck/angular-growl@0.4.0 -o "{ main: 'build/angular-growl.min.js', dependencies: { 'angular': '1.3.8'} }"
jspm install bootstrap css@0.1.0
```
But that was same as before (with jspm v0.11.0)

**Comment by guybedford** [comment]
Ok thanks, I'll post a patch out very soon.

**Comment by guybedford** [comment]
Fixed in 0.11.2. Thanks so much for letting me know about this. You'll need to run the installs again fresh.

**Comment by SekibOmazic** [comment]
Works great!  :+1:
